dkeyserver/README.md

```diff
@@ -0,0 +1,35 @@
+- If you need to synchronize the domain key between the host machine (the machine that has the domain key) and the backup machine (the machine that requests the domain key from the host machine), then you can compile first.
+```bash
```

Comment:
based on the design how to keep the domain key in sync？

and the backup machine don't provide the service to dkeycache, right?

Reply:
Q1: the invoker decides to synchronize if backup is required.
Q2: the backup machine can provide the service to dkeycache, use the same domain key as the host machine.

Reply:
Q1: the invoker decides to synchronize if backup is required.
Q2: the backup machine can provide the service to dkeycache, use the same domain key as the host machine.